README.md

```diff
@@ -24,6 +24,19 @@ npm i demodb -g
 demodb start test
 ```
 
+### Client usage
+```
+npm i demodb
+```
+```js
+const Demodb = require('demodb')
+const demodb = new Demodb('http://localhost:8081')
+
+const col = demodb.collection('users')
+await col.insert({ winter: 'is coming', tag: 'got' })
```

Comment:
GoT... winter is coming... 😅

Reply:
And the internal http client is [`got`](https://github.com/sindresorhus/got) ;)